scripts/deploy-beta-aragonid.js

```diff
@@ -1,29 +1,43 @@
 const namehash = require('eth-ens-namehash').hash
 const keccak256 = require('js-sha3').keccak_256
 
-const FIFSResolvingRegistrar = artifacts.require('FIFSResolvingRegistrar')
-const ENS = artifacts.require('AbstractENS')
-
-const owner = process.env.OWNER || '0x4cb3fd420555a09ba98845f0b816e45cfb230983'
-const ens = process.env.ENS || '0xfbae32d1cde62858bc45f51efc8cc4fa1415447e'
+const globalArtifacts = this.artifacts // Not injected unless called directly via truffle
+const defaultOwner = process.env.OWNER || '0x4cb3fd420555a09ba98845f0b816e45cfb230983'
+const defaultENSAddress = process.env.ENS || '0xfbae32d1cde62858bc45f51efc8cc4fa1415447e'
 
 const tld = namehash('eth')
 const label = '0x'+keccak256('aragonid')
 const node = namehash('aragonid.eth')
 
-module.exports = async callback => {
-  const publicResolver = await ENS.at(ens).resolver(namehash('resolver.eth'))
-  console.log('deploying AragonID')
-  const aragonID = await FIFSResolvingRegistrar.new(ens, publicResolver, node)
-
-  console.log('assigning ENS name to AragonID')
-  await ENS.at(ens).setSubnodeOwner(tld, label, aragonID.address)
-
-  console.log('assigning owner name')
+module.exports = async (
+  truffleExecCallback,
+  {
+    artifacts = globalArtifacts,
+    ensAddress = defaultENSAddress,
+    owner = defaultOwner,
+    verbose = true
+  } = {}
+) => {
+  const log = (...args) => {
+    if (verbose) { console.log(...args) }
+  }
+
+  log(ensAddress, owner)
```

Comment:
I'd prefix this message with something, like `Deploying AragonID with ENS: ... and owner: ...` as otherwise this message will look like `0x<addr> 0x<addr>` and be really confusing haha